internal/renderers/skia/textlayout.rs

```diff
@@ -97,9 +97,13 @@ pub fn create_layout(
     }
 
     style.set_text_align(match h_align {
-        items::TextHorizontalAlignment::Left => skia_safe::textlayout::TextAlign::Left,
+        TextHorizontalAlignment::Start | items::TextHorizontalAlignment::Left => {
```

Comment:
I think start should be mapped to `skia_safe::textlayout::TextAlign::Start`; same for end.